schema/pgsql/upgrades/001.sql

```diff
@@ -0,0 +1,20 @@
+DROP TABLE object_extra_tag;
+
+ALTER TABLE source DROP CONSTRAINT ck_source_icinga2_has_config;
+ALTER TABLE source DROP CONSTRAINT IF EXISTS ck_source_bcrypt_listener_password_hash;
+ALTER TABLE source ADD CONSTRAINT ck_source_bcrypt_listener_password_hash CHECK (listener_password_hash LIKE '$2_$%');
+ALTER TABLE source
+  DROP COLUMN icinga2_base_url,
+  DROP COLUMN icinga2_auth_user,
+  DROP COLUMN icinga2_auth_pass,
+  Drop COLUMN icinga2_ca_pem,
+  Drop COLUMN icinga2_common_name,
+  Drop COLUMN icinga2_insecure_tls,
+  ALTER COLUMN listener_password_hash SET NOT NULL;
```

Comment:
This column was previously nullable, and I already have an entry with a null value. This means that I cannot perform this upgrade. The following is missing:
```sql
UPDATE source SET listener_password_hash = '$2_$%' WHERE listener_password_hash IS NULL;
```